src/imports.jl

```diff
@@ -110,7 +110,7 @@ import LinearAlgebra:
   tr,
   transpose
 
-using ITensors.NDTensors.Expose: cpu
+using ITensors.NDTensors.GPUArraysCoreExtensions: cpu
```

Comment:
Because this is needed, this is technically a breaking change so we should bump the minor version of `NDTensors`.

Reply:
@kmp5VT please bump the minor version of `NDTensors` in `Project.toml` and then change `Project.toml` of `ITensors` accordingly, including bumping the patch version of `ITensors`.